### PR TITLE
Some fixes and propositions to template building

### PR DESCRIPTION
**Pull request by jevank**
Details in commit description

### CODE REVIEW

File: template_scripts/09_cleanup.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ echo "--> Rebuilding rpm database..."
 chroot `pwd`/mnt /bin/rpm --rebuilddb 2> /dev/null
 
 if [ -x mnt/usr/bin/dnf ]; then
-    chroot mnt dnf clean all
+    chroot `pwd`/mnt dnf clean all
```
Comment (fepitre) [inline template_scripts/09_cleanup.sh:13]: Prefer `$(...)` syntax instead of the deprecated backticks

**Review by fepitre — CHANGES_REQUESTED** [review]

File: template_scripts/packages_fc.list [hunk 2]
```diff
@@ -72,6 +74,7 @@ zip
 --exclude=mlocate
 --exclude=nss-mdns
 --exclude=openssh-server
+--exclude=pipewire
```
Comment (fepitre) [inline template_scripts/packages_fc.list:77]: We are thinking to use `pipewire` to replace our Pulseaudio implementation so I guess it's too early for this one.

**Review by fepitre — CHANGES_REQUESTED** [review]

File: template_scripts/09_cleanup.sh (hunk 1 above)
Comment (jevank) [reply template_scripts/09_cleanup.sh]: Agree, fixed

File: template_scripts/packages_fc.list (hunk 2 above)
Comment (jevank) [reply template_scripts/packages_fc.list]: Thanks for review, fixed

**Comment by jevank** [comment]
Should I had done force push or follow-up commit?..

**Review by marmarek — APPROVED** [review]

**Pull request merged**